etc/samples/sds_objects_generic.sample.yaml

```diff
@@ -0,0 +1,37 @@
+# This schema is not used stand-alone, it is a sub-section of "tendrl_definitions_<sds_nvr>.yaml" | "tendrl_definitions_generic.sample.yaml
+
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
+    value: "/clusters/gluster/$cluster.id/volumes/$volume.id
```

Comment:
@r0h4n we cannot have that schema for the etcd value this way, to get the details I need to know the sds_type (gluster in this example), which is not possible to know till I get the cluster details. I would prefer /clusters/$cluster.id/volumes/$volume.id with the sds_type and sds_version inside the cluster details